.github/workflows/main.yml

```diff
@@ -46,3 +46,84 @@ jobs:
       run: |
         conda activate test-microsetta-interface
         pytest
```

Comment:
Oof.  This adds a lot of dependencies that have to be kept up with any changes to private api.  I guess there's no way to just say Install Private Api With Dependencies that references whatever private api's dependencies are?

Reply:
isn't that done here by installing dependencies from the private API's requirements files which are part of its repo?